Project-Hotpot/Project-Hotpot/Managers/UserSettingsManager.swift

```diff
@@ -71,7 +71,14 @@ class UserSettingsManager: NSObject {
         }
         removedGenres.insert(genre)
         userGenres.remove(genre)
-        
+    }
+    
+    func removeMultipleGenresFromPreferences(genres: Set<String>) {
+        genres.map{ genre in removeGenreFromPrefences(genre: genre)}
```

Comment:
Nit: `genres.forEach` (https://developer.apple.com/documentation/swift/array/foreach(_:))
Also, another good opportunity to use `$0`